### DISCUSSION TITLE
Benchmark graph interpretation

### DISCUSSION THREAD

**Discussion by joangog** [ultralytics/yolov5#4505] [Q&A] [answered]
I have a question regarding the benchmark graph [here.](https://user-images.githubusercontent.com/26833433/114313216-f0a5e100-9af5-11eb-8445-c682b60da2e3.png)
What do the multiple points for a specific model mean? For example for YOLOv5s6 we can see that it is a line of 8 points instead of just one point. What adjustments to the model have been made to get the different values?
Thanks.

**Comment by glenn-jocher** [comment, marked as answer]
@joangog Figure Notes provide commands to reproduce this plot.

**Reply by joangog** [reply]
I understand how to reproduce it but can someone provide a quick explanation of what those points mean for each model?

**Reply by glenn-jocher** [reply]
@joangog evaluations over image size. See relevant region of val.py:
https://github.com/ultralytics/yolov5/blob/2da6444c9251f77cfd3e410369cd067245d961b5/val.py#L336-L350

**Reply by joangog** [reply]
Thank you very much!
